src/lib/AlestraClient.ts

```diff
@@ -1,7 +1,6 @@
 import { WebsocketClient } from '#lib/websocket/WebsocketClient';
-import { ENABLE_EVLYN } from '#root/config';
 import { SapphireClient } from '@sapphire/framework';
 
 export class AlestraClient extends SapphireClient {
-	public websocket = ENABLE_EVLYN ? new WebsocketClient(this) : null;
+	public websocket = new WebsocketClient(this);
```

Comment:
Either keep the check, or drop the line (and the class) entirely. Right now this change would force Evlyn to be up, even in development mode.

Reply:
removed the websocket client creation and the following code will be removed in a follow up PR

Reply:
added the changes in the same PR